test/DynamoCoreTests/FileReading.cs

```diff
@@ -58,11 +58,22 @@ public void Defect_MAGN_1380_dyf()
         {
             // Details steps are here: http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-1380
 
-            string testFilePath = Path.Combine(localDynamoFileTestFloder, "future_file.dyf");
+            string testFilePath = Path.Combine(localDynamoFileTestFolder, "future_file.dyf");
 
             OpenModel(testFilePath);
             Assert.AreEqual(CurrentDynamoModel.CurrentWorkspace.Nodes.Count(), 0);
             Assert.AreEqual(CurrentDynamoModel.CurrentWorkspace.Connectors.Count(), 0);
         }
+
+        [Test]
+        public void CanReadIsSelectedInputProperty()
```

Comment:
I don't know whether it's right place for this test. I didn't file with NodeModel tests.